src/test/java/marquez/api/JobEqualityTest.java

```diff
@@ -0,0 +1,69 @@
+package marquez.api;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+import java.sql.Timestamp;
+import java.time.Instant;
+import java.util.UUID;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class JobEqualityTest {
+
+  private static final UUID JOB_UUID = UUID.randomUUID();
+  private static final String JOB_NAME = "myJob";
+  private static final String OWNER_NAME = "myOwner";
+  private static final Timestamp NOMINAL_TIME = Timestamp.from(Instant.now());
+  private static final String CATEGORY = "myCategory";
+  private static final String DESCRIPTION = "the first job";
+
+  private static Job JOB;
+
+  @BeforeClass
+  public static void setupTestJob() {
+    JOB = new Job(JOB_UUID, JOB_NAME, OWNER_NAME, NOMINAL_TIME, CATEGORY, DESCRIPTION);
```

Comment:
I think you can move this out of `setupTestJob` and just initialize `JOB` to this? If you set `JOB` to be `final` then there's no chance of it getting modified.